plugins/task-resources/src/components/state/StatesEditor.svelte

```diff
@@ -83,8 +94,77 @@
   async function onAdd (_class: Ref<Class<State | DoneState>>) {
     dispatch('add', _class)
   }
+
+  async function onDescriptionChange (value: string) {
+    dispatch('descriptionChange', { value })
+  }
+
+  async function onShortDescriptionChange (value: string) {
+    dispatch('shortDescriptionChange', { value })
+  }
 </script>
 
+{#if template?.space === 'recruit:space:VacancyTemplates' && space}
```

Comment:
Please make it configurable, so you need some mixin added to space or template, so task-resources should do not know about recruit.

Reply:
Done